src/NotificationEventMailing.php

```diff
@@ -515,11 +515,12 @@ private static function attachDocuments(Email $mail, array $documents_ids)
         }
     }
 
+    #[Override]
     protected static function extraRaise($params)
     {
         //Set notification's signature (the one which corresponds to the entity)
-        $entity = $params['notificationtarget']->getEntity();
-        $params['template']->setSignature(Notification::getMailingSignature($entity));
+        $entities_id = (int) $params['notificationtarget']->getEntity();
+        $params['template']->setSignature(Notification::getMailingSignature($entities_id));
```

Comment:
I am not sure about this change. If the `getEntity()` result is `''`, I guess we have to fallback to `$CFG_GLPI['mailing_signature']` (the global GLPI config value) in `Notification::getMailingSignature()`.